### ISSUE TITLE
Package directory named test gets ignored

### ISSUE THREAD

**Issue by mikearnaldi** [folke/ultra-runner#70]
I am trying to use this to build:
https://github.com/Matechs-Garage/matechs-effect

I have a package named test, namely:
https://github.com/Matechs-Garage/matechs-effect/tree/master/packages/test

This directory gets ignored in build, if I rename the directory to something else then it's all ok.

I have worked around the issue by renaming on the ci pipeline:
https://github.com/Matechs-Garage/matechs-effect/blob/master/.github/workflows/publish.yml#L22

**Comment by folke** [comment]
That's definitily odd :-)

I'll have a look at it later today or tomorrow morning.

Thank you for reporting!

**Comment by folke** [comment]
Just quickly checked and packages in `test` are indeed excluded. Totally forgot I added a default ignore filter:

```js
const DEFAULT_IGNORE = [
  "**/node_modules/**",
  "**/bower_components/**",
  "**/test/**",
  "**/tests/**",
  "**/__tests__/**",
]
```

This should probably only be used in case no lerna, yarn, ... is detected and we do a fallback to a simple recursive search for `package.json` files.

Need to check a couple of things first before I make that change.

**Comment by mikearnaldi** [comment]
yeah that ignore should only apply to not monorepos, also I noticed that the tests in the actual packages are not ignored (they appear in changed)

**Comment by folke** [comment]
Will be fixed as part of #74

**Comment by folke** [comment]
Closing in favor of #74

**Comment by folke** [comment]
Fixed in v3.3.0
